dataflow/build.gradle

```
@@ -66,6 +66,9 @@ task manual(group: 'Documentation') {
 }
 
 tasks.withType(Test) {
+    // Disable gradle generated test task as dataflow framework does not use junit for testing and enable it will issue deprecation warnings.
```

```suggestion
    // Disable the gradle generated test task as the dataflow framework does not use JUnit for testing. If it were kept enabled (which is the default), gradlew would produce a deprecation warning.
```

How does this sound? Maybe you need to reformat.

---

Sounds good to me. Let me commit the change.